docs/faq.md

```diff
@@ -8,10 +8,22 @@ AdonisFX provides with two solvers for muscle simulation. The use of one or the
 
 In order to add activation to the muscles it is necessary to define the fibers direction. This can be achieved by: first, painting the tendon weights to generate an initial estimation of fibers direction; and second, combing the fibers through the AdonisFX Paint Tool to customise the final fibers flow along the surface. Once the muscle has a valid fiber direction at each vertex, then the activation attribute triggers the contraction of the fibers. Ultimately, the level of activation can be modulated by an AdonisFX sensor to connect the contraction of the muscle to the animation of the rig. For more information about connecting sensors to muscles, please check this [section](maya/simple_setup#connect-sensors).
 
+## How can I combine multiple activation values?
+
+Multiple activation values from different AdonisFX sensors (AdnSensorPosition, AdnSensorDistance, AdnSensorRotation) can be combined into a single value using the **AdnActivation** node. This node allows to override, add, subtract, multiply or divide multiple input activations to compute a single output value. For more details about the **AdnActivation** node, please refer to the [AdnActivation](maya/activation.md) page.
```

Comment:
```suggestion
Multiple activation values from different AdonisFX sensors (AdnSensorPosition, AdnSensorDistance, AdnSensorRotation) can be combined into a single value using the **AdnActivation** node. This node allows overriding, adding, subtracting, multiplying and dividing input activations to compute a single output value. For more details about the **AdnActivation** node, please refer to the [AdnActivation](maya/activation.md) page.
```

Reply:
If you finally apply this change please also add it to the AdnActivation page

Reply:
Done.